### ISSUE TITLE
Images

### ISSUE THREAD

**Issue by ghost** [andrenes/prj-rev-bwfs-dasmoto#4]
https://github.com/andrenes/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/index.html#L13

Give your images an `alt` attribute with a description of the image. 

This helps ensure your website's accessibility when it's unable to load images.

You should also save these images into your local project folder and use a relative file path to them instead of a URL, just in case the provided URLs ever break.
